### DISCUSSION TITLE
Favorites list

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#123] [Q&A] [answered]
New favorites list coming in a next Cruiser (see video):
- Drag and drop: reorder favorites
(drag handler is the icon)
- Swipe left / right: delete favorites
- Long press: menu
  - Edit
  - Category / Color
  - Show on map
  - Select (opens actions menu)
  - Sort

[device-2023-04-08-191325.webm](https://user-images.githubusercontent.com/3484020/230731841-36af66d2-8954-4926-bde3-0198690186c3.webm)

**Comment by DolomitesTom** [comment]
Good prospects

**Comment by ramondevic** [comment]
Being able to define the color and icon would be very interesting and would help a lot in the visualization and not just seeing an orange round with a white star inside.

**Reply by devemux86** [reply]
@ramondevic this is already in active development, see https://github.com/devemux86/cruiser/discussions/45.

**Comment by ramondevic** [comment]
![imatge](https://user-images.githubusercontent.com/122691401/233079578-0745aef7-c78b-4731-a104-f1416f729f75.png)

**Comment by devemux86** [comment, marked as answer]
New favorites list with drag & drop is available in [Cruiser 3.0.18](https://play.google.com/store/apps/details?id=gr.talent.cruiser).

For instructions please see the announcement:
- https://github.com/devemux86/cruiser/discussions/123#discussion-5058829

Please let me know how you see it!

---

Favorites development continues with more new features:
- https://github.com/devemux86/cruiser/discussions/45

**Reply by DolomitesTom** [reply]
Absolute great, edit, show on map, sort by name, sort by distance, last is cool without end

**Comment by WolfgangZ42** [comment]
Using my favorites mainly driving with my cars, all favorites are established around Munich. So sort by name is sufficient. 
Just an idea to improve the handling a little bit:
<img width="200" src="https://user-images.githubusercontent.com/121806238/235112472-7470a124-d5f9-4ef9-bbcc-fb2dab0c6802.png"/>   <img width="200" src="https://user-images.githubusercontent.com/121806238/235112518-9eb8fdc1-5a06-44e2-bc23-edb80b7e5851.png"/>   <img width="200" src="https://user-images.githubusercontent.com/121806238/235112496-0decd143-576b-45f1-a5bf-f76da92a9a50.png"/>   
Perfect if that would work even if "Show favorites on map" is off.

**Reply by devemux86** [reply]
@WolfgangZ42 thanks for the suggestion!

You probably mean an option to open automatically the actions menu of the favorite?

**Reply by devemux86** [reply]
The "Select" action will be available in the next Cruiser update.

We can also add it in the new waypoints list:
- https://github.com/devemux86/cruiser/discussions/114#discussioncomment-5559274
